src/react/utils/index.js

```diff
@@ -1,3 +1,18 @@
+export const systemMetadataKeys = ['CacheControl', 'ContentDisposition',
```

Comment:
I would create this dynamically using something like `systemMetadata.map(d => d.key)`